pkgs/applications/science/machine-learning/sc2-headless/maps.nix

```diff
@@ -1,11 +1,48 @@
 { fetchzip
 }:
-
+let
+  fetchzip' = args: (fetchzip args).overrideAttrs (old: { UNZIP = "-P iagreetotheeula"; });
```

Comment:
is the eula the same as the license for sc2?

Reply:
Yep